doc/source/reference/routines.char.rst

```diff
@@ -6,8 +6,14 @@ String operations
 .. module:: numpy.char
 
 The `numpy.char` module provides a set of vectorized string
-operations for arrays of type `numpy.str_` or `numpy.bytes_`.
-All of them are based on the string methods in the Python standard library.
+operations for arrays of type `numpy.str_` or `numpy.bytes_`. For example
+
+      >>> np.char.capitalize("numpy")
+      array('Numpy', dtype='<U5')
```

Comment:
I'd vote to use a list/array of strings in this example, if only because I think the best-practice would be to use the string methods directly when working with individual strings. Maybe something like:

```suggestion
      >>> np.char.capitalize(["python", "numpy"])
      array(['Python', 'Numpy'], dtype='<U6')
```